qiime2/sdk/tests/test_plugin_manager.py

```diff
@@ -60,6 +60,42 @@ def test_importable_types(self):
 
     # TODO: add tests for type/directory/transformer registrations
 
+    def test_importable_formats(self):
+        _format = self.pm.importable_formats
+        exp = {'IntSequenceV2DirectoryFormat':
+               FormatRecord(format='qiime2.plugin.model.directory_format.'
+                                   'IntSequenceV2DirectoryFormat',
+                            plugin=self.plugin),
+               'FourIntsDirectoryFormat':
+               FormatRecord(format='qiime2.core.testing.format.'
+                                   'FourIntsDirectoryFormat',
+                            plugin=self.plugin),
+               'IntSequenceDirectoryFormat':
+               FormatRecord(format='qiime2.plugin.model.directory_format.'
+                                   'IntSequenceDirectoryFormat',
+                            plugin=self.plugin),
+               'IntSequenceFormat':
+               FormatRecord(format='qiime2.core.testing.format.'
+                                   'IntSequenceFormat',
+                            plugin=self.plugin),
+               'IntSequenceFormatV2':
+               FormatRecord(format='qiime2.core.testing.format.'
+                                   'IntSequenceFormatV2',
+                            plugin=self.plugin),
+               'MappingDirectoryFormat':
+               FormatRecord(format='qiime2.core.testing.format.'
+                                   'MappingDirectoryFormat',
+                            plugin=self.plugin),
+               'MappingFormat':
+               FormatRecord(format='qiime2.core.testing.format.MappingFormat',
+                            plugin=self.plugin),
+               'SingleIntFormat':
+               FormatRecord(format='qiime2.core.testing.format.'
+                                   'SingleIntFormat',
+                            plugin=self.plugin)}
+
+        self.assertTrue(set(exp).issubset(set(_format)))
```

Comment:
Our worry is that then it will have to be updated if new canonical formats are added.   Is this a risk?  Or desirable?